lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -48,7 +49,7 @@ function abortIt(signal) {
   return new AbortError(undefined, { cause: signal.reason });
 }
 
-const SUPPORTED_TIMERS = ['setTimeout', 'setInterval'];
+const SUPPORTED_TIMERS = ['setTimeout', 'setInterval', 'Date'];
```

Comment:
Date should be mocked as clear functions (as sinon does) 
see: 
> var clock = sinon.useFakeTimers();
Causes Sinon to replace the global setTimeout, clearTimeout, setInterval, clearInterval, setImmediate, clearImmediate, process.hrtime, performance.now(when available) and Date with a custom implementation which is bound to the returned clock object.
https://sinonjs.org/releases/v15/fake-timers/